assets/js/modules/search-console/components/dashboard/GoogleSitekitSearchConsoleDashboardWidget.js

```diff
@@ -58,13 +59,23 @@ const GoogleSitekitSearchConsoleDashboardWidget = () => {
 	const [ loading, setLoading ] = useState( true );
 	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
+	const isDomainProperty = useSelect( ( select ) => select( STORE_NAME ).isDomainProperty() );
+	let referenceSiteURL = useSelect( ( select ) => select( CORE_SITE ).getReferenceSiteURL() );
+	if ( referenceSiteURL.endsWith( '/' ) ) {
+		// remove the trailing slash
+		referenceSiteURL = referenceSiteURL.slice( 0, -1 );
+	}
```

Comment:
Since we're using this in multiple places now, I think it would make sense to add a simple `untrailingslashit` utility function, almost similar to what WP core has in PHP. To keep our code simple, let's in that function only strip the trailing slash and even do the check if the passed value is a string. This way we can just run the return value of the `getReferenceSiteURL` selector through it immediately, i.e. we can use a `const` for `referenceSiteURL` above.

Reply:
I added this function to `assets/js/util/sanitize.js` as it seemed fitting to me. Please let me know if you have a better place in mind.